### PR TITLE
Update RHEL suffix to rhel9 from rhel8

### PR DESCRIPTION
**Pull request by zkayyali812**
Also removes emea ops image, as this is no longer necessary and it is not being published.
https://issues.redhat.com/browse/AAP-14163

This issue updates our image reference in our documentation to be our expected image reference once published

### CODE REVIEW

**Review by chrisahl — APPROVED** [review]
I assume you did a global search/replace.  I did not try and search the whole repo.  In the future I wonder if we should make this a variable and use {{ }}.  Then we just have to change in one place.

And wonder if we should not merge until we are 100% sure the boutique tag is correct and published?

**Comment by thenets** [comment]
Great :+1: 
Let's wait the [registry.redhat.io/ansible-on-clouds/ansible-on-clouds-ops-rhel9:2.4.20230630](http://registry.redhat.io/ansible-on-clouds/ansible-on-clouds-ops-rhel9:2.4.20230630) be available. Then we can merge it.

**Comment by zkayyali812** [comment]
Thanks for the reviews! 

I agree with the above comments.

@ianf77 Please feel free to review this PR and even approve if you think it is good, but let us not merge this in until we have validated our image reference is as expected.

**Comment by zkayyali812** [comment]
@chrisahl That is a great idea!! One that we should add to our next doc sync Id expect

**Comment by schmidtd** [comment]
Pull is good
```
$ podman pull registry.redhat.io/ansible-on-clouds/ansible-on-clouds-ops-rhel9:2.4.20230630 
Trying to pull registry.redhat.io/ansible-on-clouds/ansible-on-clouds-ops-rhel9:2.4.20230630...
Getting image source signatures
Checking if image destination supports signatures
Copying blob sha256:8519e3a3b57949af0c943d0e3c097be505926896ee9ba97314cb240ecc359e82
...
```

**Review by schmidtd — APPROVED** [review]
LGTM

**Comment by chrisahl** [comment]
We can pull image and use it so this can be merged

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**
